src/Controller/UI/RemotelyHostedDatasetsController.php

```diff
@@ -90,7 +90,7 @@ public function postAction(Request $request)
             if ($datasetSubmission instanceof DatasetSubmission) {
                 $preReqs = $this->remotelyHostedPrerequsiteCheck($datasetSubmission);
                 if (count($preReqs) > 0) {
-                    $message = implode(', ', $preReqs);
+                    $message = 'CONFIRM:' . implode(', ', $preReqs);
```

Comment:
This is not a great way to do this. Maybe change the response HTTP code on line L113 to something else, 400 ish or so.